scripts/single-node.sh

```diff
@@ -20,11 +20,9 @@ celestia-appd collect-gentxs
 # Set proper defaults and change ports
 # If you encounter: `sed: -I or -i may not be used with stdin` on MacOS you can mitigate by installing gnu-sed
 # https://gist.github.com/andre3k1/e3a1a7133fded5de5a9ee99c87c6fa0d?permalink_comment_id=3082272#gistcomment-3082272
-sed -i'.bak' 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' ~/.celestia-app/config/config.toml
-sed -i'.bak' 's/timeout_commit = "25s"/timeout_commit = "1s"/g' ~/.celestia-app/config/config.toml
-sed -i'.bak' 's/timeout_propose = "3s"/timeout_propose = "1s"/g' ~/.celestia-app/config/config.toml
-sed -i'.bak' 's/index_all_keys = false/index_all_keys = true/g' ~/.celestia-app/config/config.toml
-sed -i'.bak' 's/mode = "full"/mode = "validator"/g' ~/.celestia-app/config/config.toml
+sed -i'.bak' 's#"tcp://localhost:26657"#"tcp://0.0.0.0:26657"#g' ~/.celestia-app/config/client.toml
+sed -i'.bak' 's/timeout_commit = "10s"/timeout_commit = "1s"/g' ~/.celestia-app/config/config.toml
```

Comment:
[not for reviewers]
the defaults are now `10s`

Reply:
forgot to mention that https://github.com/celestiaorg/celestia-app/pull/1386#issuecomment-1433109353 changing these won't do anything. The only way to spin up a faster chain is programmatically.

Reply:
I see, thanks :+1:

Reply:
so we can just delete these `sed` commands?

Reply:
What about the config files that still contain those parameters, probably they should be deleted to avoid any confusion for validators, no?

Reply:
> so we can just delete these sed commands?

[0c8806d](https://github.com/celestiaorg/celestia-app/pull/1386/commits/0c8806d9ed2932e30f6f24af07c00cba14da5175)